runtime/src/migrations.rs

```diff
@@ -0,0 +1,162 @@
+pub(crate) mod account_data_migration {
```

Comment:
Oh, I didn't know you did this weird thing. Do you mind pulling this mod definition up to `lib.rs` and rename this file?